drivers/d3d12/rendering_device_driver_d3d12.cpp

```diff
@@ -2081,8 +2082,16 @@ dxil_validator *RenderingDeviceDriverD3D12::_get_dxil_validator_for_current_thre
 	print_verbose("Creating DXIL validator for worker thread index " + itos(thread_idx));
 #endif
 
-	dxil_validator *dxil_validator = dxil_create_validator(nullptr);
-	CRASH_COND(!dxil_validator);
+	dxil_validator *dxil_validator = nullptr;
+#if defined(_GAMING_XBOX_SCARLETT) || defined(_GAMING_XBOX)
+	dxil_validator = dxil_create_validator(nullptr);
+#else
+	String exec_path = OS::get_singleton()->get_executable_path().get_base_dir();
+	String exec_arch_path = exec_path.path_join(Engine::get_singleton()->get_architecture_name());
+	if (FileAccess::exists(exec_path.path_join("dxil.dll")) || FileAccess::exists(exec_arch_path.path_join("dxil.dll"))) {
+		dxil_validator = dxil_create_validator(nullptr);
+	}
+#endif
```

Comment:
Note: this can be replaced with the single line after https://github.com/godotengine/godot-nir-static/pull/8
```c++
dxil_validator *dxil_validator = dxil_create_validator(nullptr);
```